src/main/java/az/edu/strangers/RoboCat.java

```diff
@@ -0,0 +1,14 @@
+package az.edu.strangers;
+
+public class RoboCat extends Pet {
+
+    public RoboCat(String nickName, Integer age, Integer trickLevel, String[] habits) {
+        super(Species.CAT, nickName, age, trickLevel, habits);
```

Comment:
Species.CAT shouldn't be in super's constructor. It should be in the class's constructor